src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: (a, b) => a + b,
```

Comment:
The `add` method should update the `result` property of the calculator object. Consider using the `operate` method to perform this operation and update the result.